app/src/composables/use-item.ts

```diff
@@ -120,12 +120,20 @@ export function useItem(
 		saving.value = true;
 		validationErrors.value = [];
 
-		const errors = validateItem(
-			merge({}, defaultValues.value, item.value, edits.value),
-			fieldsWithPermissions.value,
-			isNew.value
+		const payloadToValidate = mergeWith(
+			{},
+			defaultValues.value,
+			item.value,
+			edits.value,
+			function (from: any, to: any) {
+				if (typeof to !== undefined) {
```

Comment:
`typeof to !== undefined` will always evaluate to true as `typeof` always returns a string.

Reply:
Nice catch 🤝